packages/monitoring-system/metrics-dashboard/scripts/deploy.sh

```diff
@@ -0,0 +1,17 @@
+# build and deploy the dashboard from a fresh clone
```

Comment:
Needs license header

Reply:
good catch -- I guess support for `.sh` files still needs to be added to the license checker bot